packages/search/lib/dropElasticsearch.js

```diff
@@ -1,9 +1,13 @@
+let debug = console.log
 const Elasticsearch = require('@orbiting/backend-modules-base/lib/Elasticsearch')
 
-const drop = async (prefix) => {
+const drop = async (prefix, doDebug) => {
   if (!prefix) {
     throw new Error("can't drop, prefix not specified")
   }
+  if (doDebug !== undefined && !doDebug) {
+    debug = identity => identity
```

Comment:
If you call this function as follows, the result will not be as expected:

```
debug(prefix1, false)
debug(prefix2)
```

Maybe we could just do the following:

```
const debug = doDebug === false ? i => i : console.log
```

Reply:
also, why does it need the identy function? `console.log` returns `undefined`.

Reply:
re identity, do you have another suggestion? Would you like `a => false` better?

Reply:
`() => {}`, or `function () {}`